assets/packages/limesurvey/survey.js

```diff
@@ -61,7 +61,7 @@ function triggerEmRelevanceSubQuestion(){
         data = $.extend({style:'hidden'}, data);
         $(this).removeClass("ls-irrelevant ls-"+data.style);
         if(data.style=='disabled'){
-            $(event.target).find('input').each(function(itrt, item ){
+            $(event.target).find('input:not(.total)').each(function(itrt, item ){
```

Comment:
Maybe `$(event.target).find('.answer-item input)')` 
I really dislike to use `total` class only … create a new specific class, or apply only to needed input.